front/src/modules/people/components/PeoplePicker.tsx

```diff
@@ -29,16 +34,24 @@ export function PeoplePicker({
     relationPickerSearchFilterScopedState,
   );
 
+  const currentEditableFieldEntityId = useContext(EditableFieldEntityIdContext);
```

Comment:
the PeoplePicker should not be aware of EditableFieldEntityIdContext as it also can be used without this context. 
EditableFieldEntityIdContext should be used as EditableField level and and company should be passed as an option props to PeoplePicker